testutils/testutils.go

```diff
@@ -0,0 +1,55 @@
+package testutils
+
+import (
+	"context"
+	"database/sql"
+	_ "modernc.org/sqlite"
+	"testing"
+)
+
+func GetEmptyDb(t *testing.T) *sql.DB {
+	t.Helper()
+	db, err := sql.Open("sqlite", ":memory:")
+	FailOnError(err, t)
+	return db
+}
+
+func ExpectError(err error, t *testing.T) {
+	t.Helper()
+	ExpectSuccess(err != nil, t)
+}
+
+func ExpectErrorWithMessage(err error, msg string, t *testing.T) {
+	t.Helper()
+	ExpectError(err, t)
+	if err.Error() != msg {
+		t.Fatalf("Expected error '%s', but got '%s' instead.", msg, err.Error())
+	}
+}
+
+func FailOnError(err error, t *testing.T) {
+	t.Helper()
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func ExpectFailure(cond bool, t *testing.T) {
```

Comment:
In other languages, we call this `assertFalse` (and `assertTrue` respectively), just in case you're looking for naming suggestions ;)

Reply:
Sure, but with those names, you wouldn't get these almost philosophical snippets: 
```go
func ExpectSuccess(cond bool, t *testing.T) {
	t.Helper()
	ExpectFailure(!cond, t)
}
```